examples/rabbitmq-hub/README.md

```diff
@@ -1,5 +1,12 @@
 # A Websub Hub Based on RabbitMQ
 
+[![Star on Github](https://img.shields.io/badge/-Star%20on%20Github-blue?style=social&logo=github)](https://github.com/ballerina-platform/module-ballerinax-rabbitmq)
+
+_Authors_: @aashikam      
+_Reviewers_: @shafreenAnfar  
+_Created_: 2021/08/30
```

Comment:
```suggestion
_Created_: 2021/08/30  
```

Reply:
Without these spaces, both `created` & `updated` fields appeared in the same line.